_overviews/scala3-book/taste-hello-world.md

```diff
@@ -47,53 +47,27 @@ object hello {
 {% endtabs %}
 <!-- End tabs -->
 
-Next, compile the code with `scalac`:
+Next, compile and run the code with `scala`:
 
 ```bash
-$ scalac hello.scala
+$ scala run hello.scala
 ```
 
-If you’re coming to Scala from Java, `scalac` is just like `javac`, so that command creates several files:
+When you run the command for the first time, two hidden directories will be created: `.bsp` and `.scala-build`. The first
+one contains the [Build Server Protocol](https://build-server-protocol.github.io/) information for IDEs, and the second one contains the results
+of compilation.
```

Comment:
I don't think this is useful information for a user at this stage of their learning. Consider removing that paragraph.